src/Routes.tsx

```diff
@@ -122,6 +132,26 @@ const routes: RouteType[] = [
     exact: true,
     component: Lighthouse,
   },
+  {
+    path: routesEnum.besu,
+    exact: true,
+    component: Besu,
+  },
+  {
+    path: routesEnum.nethermind,
+    exact: true,
+    component: Nethermind,
+  },
+  {
+    path: routesEnum.erigon,
+    exact: true,
+    component: Erigon,
+  },
+  {
+    path: routesEnum.geth,
+    exact: true,
+    component: Geth,
+  },
```

Comment:
Suggest using alphabetical order. :)

Reply:
For sure... placed them in reverse order of current usage, but certain not set on this. We could also randomize these if people are interested, but can default to abc